File: CS/CodeBehind/XPO/ServerMode/EditIssueInfo.cs
using DevExpress.Mvvm;
using System.Collections;
using XPOIssues.Issues;

namespace XPOIssues {
    public class EditIssueInfo : BindableBase {
        public EditIssueInfo(DevExpress.Xpo.UnitOfWork unitOfWork, IList users) {
            UnitOfWork = unitOfWork;
            Users = users;
        }
        public DevExpress.Xpo.UnitOfWork UnitOfWork { get; }
        public IList Users { get; }
    }
}

File: VB/CodeBehind/XPO/InstantFeedbackMode/EditIssueInfo.vb
Imports DevExpress.Xpo
Imports DevExpress.Mvvm
Imports System.Collections.Generic
Imports DevExpress.Xpf.Grid
Imports XPOIssues.Issues

Public Class EditIssueInfo
    Inherits BindableBase

    Public Sub New(ByVal unitOfWork As UnitOfWork, ByVal users As IList)
        Me.UnitOfWork = unitOfWork
        Me.Users = users
    End Sub

    Public ReadOnly Property UnitOfWork As UnitOfWork
    Public ReadOnly Property Users As IList
End Class
